subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/StaleOutputIntegrationTest.groovy

```diff
@@ -134,7 +134,7 @@ class StaleOutputIntegrationTest extends AbstractIntegrationSpec {
         when:
         succeeds("myTask")
         then:
-        dirInBuildDir.assertDoesNotExist()
+        dirInBuildDir.assertIsDir()
```

Comment:
❌ For this test to test anything, we would need to create a file in the `dirInBuildDir` and then make sure it has been removed by stale output file removal.

Reply:
Added some stale files.